torch_dicom/preprocessing/resize.py

```diff
@@ -13,6 +13,9 @@
 from .crop import E
 
 
+EPS: Final = 1e-6
```

Comment:
Instead of defining a constant, can we use something like this? `torch.finfo(torch.float32).eps`